mne/viz/_3d.py

```diff
@@ -2454,8 +2454,8 @@ def _check_views(surf, views, hemi, stc=None, backend=None):
     if (views == ['flat']) ^ (surf == 'flat'):  # exactly only one of the two
         raise ValueError('surface="flat" must be used with views="flat", got '
                          f'surface={repr(surf)} and views={repr(views)}')
-    _check_option('hemi', hemi, ('split', 'both', 'lh', 'rh', 'vol'))
-    use_hemi = 'lh' if hemi == 'split' else hemi
+    _check_option('hemi', hemi, ('split', 'both', 'lh', 'rh', 'vol', None))
+    use_hemi = 'lh' if hemi == 'split' or hemi is None else hemi
```

Comment:
Can either revert this and related `Brain(..., hemi=None)` changes, or we need to add a test that uses `hemi=None` in `test_brain.py`

Reply:
Ok, a test should be easy